frontend/craco.config.js

```diff
@@ -1,10 +1,43 @@
 const path = require('path');
 
+const WorkboxWebpackPlugin = require('workbox-webpack-plugin');
+
+const isEnvProduction = process.env.NODE_ENV === "production"
+
 module.exports = {
-  // Output to ./course (instead of ./build)
   webpack: {
     configure: (webpackConfig, {env, paths}) => {
+      // Output to ./course (instead of ./build)
       paths.appBuild = webpackConfig.output.path = path.resolve('course');
+
+      // Override config for WorkboxWebpackPlugin.{GenerateSW,InjectManifest}
+      webpackConfig.plugins = webpackConfig.plugins.map(plugin => {
+        if(plugin.constructor.name === 'InjectManifest') {
+          console.info({plugin})
+          return new WorkboxWebpackPlugin.InjectManifest({
```

Comment:
So, doing the `WorkboxWebpackPlugin` override this way actually gets past that error I was seeing (the one about missing `self.__WB_MANIFEST`). So that's good.